games/warzone2100/Portfile

```diff
@@ -38,16 +38,15 @@ if {$subport eq ${name}} {
     PortGroup           cmake 1.1
     PortGroup           legacysupport 1.1
 
-    github.setup        Warzone2100 ${name} 4.2.7
-    revision            1
```

Comment:
please leave the "revision 0" line

Reply:
I personally disagree with the decision this past year to have everyone keep "revision 0" in our Portfiles, even though that is the default. But I have gone ahead and added it in, as you have asked.